src/components/welcome/WelcomeSection.vue

```diff
@@ -0,0 +1,130 @@
+<script>
+import { defineComponent } from "vue";
+import SectionContainer from "@/components/global/SectionContainer.vue";
+import SectionTitlePrimary from "@/components/global/SectionTitlePrimary.vue";
+import SectionDescription from "@/components/global/SectionDescription.vue";
+import BaseButton from "@/components/global/BaseButton.vue";
+import BaseImage from "@/components/global/BaseImage.vue";
+
+export default defineComponent({
+  name: "WelcomeSection",
+  components: {
+    SectionContainer,
+    SectionTitlePrimary,
+    SectionDescription,
+    BaseButton,
+    BaseImage,
+  },
+  data() {
+    return {
+      userLogged: false,
+    };
+  },
+  computed: {
+    routeName() {
+      return this.userLogged ? { name: "BookTicket" } : { name: "UserLogIn" };
+    },
+  },
+});
+</script>
+
+<template>
+  <section class="welcome">
+    <SectionContainer class="welcome__container">
+      <div class="welcome__text">
+        <SectionTitlePrimary
+          title="Welcome to the best cinema in Monterail"
+          class="welcome__title"
+        />
+        <SectionDescription
+          description="Best movies, best experiences, best people. And all that doesn’t exist. Yikes."
+          class="welcome__description"
+        />
+        <BaseButton
+          class="welcome__link"
+          size="large"
+          :to="routeName"
+          colorTheme="accent-filled"
+          >Book a ticket</BaseButton
+        >
+      </div>
+      <div class="welcome__image">
+        <BaseImage
+          src="/src/assets/images/cinema.jpeg"
+          alt="spectators in the cinema hall"
+          class="welcome"
+        />
+      </div>
+    </SectionContainer>
+  </section>
+</template>
+
+<style lang="scss" scoped>
+.welcome {
+  width: 100%;
+  height: 562px;
+  @include mediumScreen {
+    height: 618px;
+  }
+}
+.welcome__container {
+  height: 100%;
+  width: 100%;
+  background-color: $colorRedWispPink;
+  @include mediumScreen {
+    background-color: $colorWhiteSnow;
+    height: 100%;
```

Comment:
Will it do anything? Considering we have `height: 100%` defined without a media query anyway?